src/realm/query_expression.hpp

```diff
@@ -251,6 +251,51 @@ struct Pow {
     typedef T type;
 };
 
+template <class T>
+struct Size {
+    int64_t operator()(T) const
+    {
+        return 1;
+    }
+    typedef T type;
```

Comment:
If you make a query for size on a integer column, then this is needed. This kind of query are, however pretty trivial as all entries have size 1.